### DISCUSSION TITLE
[2024/04/30] 백엔드 회의

### DISCUSSION THREAD

**Discussion by asd42270** [10aeat/10aeat_Backend#21] [백엔드 회의록]
### 공통 클래스 설정 PR 리뷰
- 리뷰 후 머지 완료

### Security 설정 역할분담
- @asd42270 , @Galmaeki

### 로그백 설정
- @ghrltjdtprbs 

## 배포
- 이거는 꼭 공부해오고, 고려할 사항 댓글로 달아주세요! (~수요일 오후 4시)

**Comment by Galmaeki** [comment]
- 배포 할 때 DB를 어느 서버에서 돌릴지
  - Redis를 어플리케이션과 같은 인스턴스에서 돌릴지, 별개의 인스턴스를 사용 할 지
- 어플리케이션을 몇 개나 실행시킬지
  - 한 인스턴스에서 애플리케이션을 여러개
  - 각각 다른 인스턴스에서 어플리케이션을 1개씩
    - DB를 같은 DB를 바라보게 할 지
- 프론트 어플리케이션도 백엔드 망에 추가할지
  - 백엔드 서버 접속을 프론트에서 내부망을 통해서만 하게 할지
- 어떤 브랜치에 GitHub Action을 적용 할지
